mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb

```diff
@@ -0,0 +1,502 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Install necessary python packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip3 install -U google-cloud-aiplatform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip3 install google-cloud-storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parameter settings"
```

Comment:
shouldn't we mention here some prerequisites that we except to have a GCP project and a model already in a bucket?